translations/round_structure.tex/de.po

```diff
@@ -77,6 +78,27 @@ msgid ""
 "Keep playing new Rounds until any of the Scenario's ending conditions have been met.\n"
 "\n"
 msgstr ""
+"Das Spiel ist in \\pagetarget{Rounds}{Runden} gegliedert, in dem jeder Spieler im Uhrzeigersinn, beginnend mit dem Startspieler, seinen eigenen Spielzug ausführt.\n"
+"Während ihrer Züge bewegen die Spieler ihre \\pagelink{Heroes}{Helden} auf dem Spielplan, errichten neue Gebäude in ihrer \\pagelink{Town}{Stadt} und rekrutieren \\pagelink{Units}{Einheiten} in dem Versuch, die Siegbedingungen des Szenarios zu erfüllen.\\par\n"
+"Die folgenden Schritte werden bei \\textbf{Rundenbeginn jeder Runde} außer der ersten durchgeführt:\n"
+"\\begin{itemize}\n"
+"  \\item Drehe alle zuvor verwendeten Gebäude-, Bevölkerungs- und/oder Zauberbuchmarker zurück auf ihre aktive Seite.\n"
+"  \\item Drehe alle zuvor genutzten Bewegungsmarker  zurück auf ihre aktive, grüne Seite.\n"
+"  \\item Stelle Kronen für genutzte Experteneffekte wieder her \\svgeven{expert}.\n"
+"\\end{itemize}\n"
+"Dann erhalten die Spieler je nach aktueller Rundenzahl entweder Ressourcen oder handeln eine \"Astrologen verkünden\"-Karte ab:\n"
+"\\Anfang{itemize}\n"
```

Comment:
```suggestion
"\\begin{itemize}\n"
```
Was this done by AI? 😆

Reply:
Oh no, there are more mistakes, than i thought. i think it was AI. Regulary i dont use it, but there was a situation where i accidentally pressed a key combination i didn't know and it translated everything :D i thought i had undone that. i'll do it again. :-D